payment/app.py

```diff
@@ -95,6 +94,17 @@ def watch(service_id):
     return jsonify(), 200
 
 
+@app.route('/whitelist', methods=['POST'])
+def whitelist():
+    print(request.get_json())
+    print(type(request.get_json()))
+    whitelist_request = WhitelistRequest(request.get_json())
```

Comment:
I recently added a call to `.validate()` on the schematics object to make sure the request is valid